Makefile

```diff
@@ -41,7 +41,7 @@ generate: build # Generate CLI docs and samples
 	go run ./hack/generate/samples/generate_testdata.go
 
 .PHONY: bindata
-OLM_VERSIONS = 0.16.1 0.15.1
+OLM_VERSIONS = 0.16.1 0.15.1 0.17.0
```

Comment:
Adding a new OLM version doesn't seem necessary to use the go sample in e2e tests.

Reply:
this PR is blocked by #4242. 
Without update OLM we are unable to move forward here.  The test https://github.com/operator-framework/operator-sdk/blob/master/test/e2e-go/e2e_go_olm_test.go#L55-L62 fails.

Reply:
You’re saying that OLM 0.15.1 doesn’t work with our Go sample? Then we have a bigger problem.